src/main/java/mate/academy/internetshop/dao/UserDao.java

```diff
@@ -14,4 +15,8 @@
     boolean deleteById(Long userId);
 
     boolean delete(User user);
+
+    User login(String login, String password) throws AuthenticationException;
```

Comment:
```suggestion
    User login(String login);
```
про це писала в чат групи

Reply:
ok